### PR TITLE
[ATL-1495] Add CSI alerts and remedies

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
## Purpose
[ATL-1495](https://jira.cec.lab.emc.com/browse/ATL-1495)

* Add CSI issues and remedies in ecs-cluster charts

## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
```
helm install ecs-cluster ./ecs-cluster --set global.registry=asdrepo.isus.emc.com:8099 --set storageServer.persistence.size=200Gi --set performanceProfile=Small --set tag=3.7.0.0-126744.88186424f4d  --set provision.image.tag=0.52.0.0-551.abf9c7c  --set storageClassName=baremetal-csi-sc --set fabricProxy.image.tag=green --set managementGateway.image.tag=shippable --set zookeeper.image.tag=shippable --set managementGateway.service.type=LoadBalancer --set s3.service.type=LoadBalancer --set diagnostic.service.type=LoadBalancer --set provision.enabled=True --set fluentbitAgent.image.tag=stable --set syslogAgent.image.tag=0.50.0 --set deosEvents.image.tag=0.50.0 --set accessKey=fHyMcqOmAgAD   --set storageServer.persistence.protected=True  --set global.monitoring_tag=3.7.0.0-987.5c52f226 --set global.monitoring.enabled=false --set features.enableAdvancedStatistics=false --set global.monitoring_registry=asdrepo.isus.emc.com:8099 --set global.rsyslog_enabled=false
...
NAME                                                   READY   STATUS             RESTARTS   AGE
atlas-operator-7578c8d498-psgff                        1/1     Running            0          12m
baremetal-csi-controller-6b686b54b7-fl5p6              3/3     Running            0          9m54s
baremetal-csi-node-766r5                               4/4     Running            0          9m54s
baremetal-csi-node-lf79m                               4/4     Running            0          9m54s
baremetal-csi-node-lpzs2                               4/4     Running            0          9m54s
ecs-cluster-atlas-0                                    1/1     Running            0          6m6s
ecs-cluster-cm-69dd7d654f-2gwrw                        2/2     Running            0          3m11s
ecs-cluster-control-7d48bff5f8-xfp2z                   2/2     Running            0          3m11s
ecs-cluster-diag-57d7856cd7-cpr5v                      2/2     Running            0          3m11s
ecs-cluster-event-56d7dcb68d-kpg95                     2/2     Running            0          3m11s
ecs-cluster-geo-f4499b8d6-q8wq8                        2/2     Running            0          3m11s
ecs-cluster-management-gateway-68c45c4bd9-gftpb        1/1     Running            0          3m11s
ecs-cluster-metering-68bd467866-zq58m                  2/2     Running            0          3m11s
ecs-cluster-objectsvc-7d6cc7dffb-g7v42                 2/2     Running            0          3m11s
ecs-cluster-resource-86bf7ffc5c-66745                  2/2     Running            0          3m10s
ecs-cluster-rm-77b7c6f89-j8v4l                         2/2     Running            0          3m10s
ecs-cluster-s3-554b6f5c75-xlrrb                        2/2     Running            0          3m11s
ecs-cluster-sr-767c5bdc9f-tprmh                        2/2     Running            0          3m10s
ecs-cluster-ss-0                                       1/1     Running            0          6m6s
ecs-cluster-ss-1                                       1/1     Running            0          6m6s
ecs-cluster-ss-2                                       1/1     Running            0          6m6s
ecs-cluster-ss-3                                       1/1     Running            0          6m6s
ecs-cluster-ssm-585bd89997-8k9zn                       1/2     Running            0          3m9s
ecs-cluster-zookeeper-0                                1/1     Running            0          6m6s
ecs-cluster-zookeeper-1                                1/1     Running            0          5m8s
ecs-cluster-zookeeper-2                                1/1     Running            0          4m12s
ecs-flex-fluxd-7fcb6dc7b6-lg6wm                        1/1     Running            0          12m
ecs-flex-grafana-68b8568d46-cq7mq                      2/2     Running            0          12m
ecs-flex-influxdb-operator-69944698d9-zz9hf            1/1     Running            0          12m
ecs-flex-prometheus-alerts-7c7476d-5n7jm               2/2     Running            0          12m
ecs-flex-service-pod-8b9fb5f4c-qkdz7                   1/1     Running            0          12m
ecs-flex-statefuldaemonset-operator-84bbf75655-4xnmj   1/1     Running            0          12m
ecs-flex-telegraf-8cf668598-bf5pp                      1/1     Running            0          12m
ecs-flex-throttler-67ccb75865-cz8lm                    1/1     Running            0          12m
objectscale-operator-cb6c78bd7-q477w                   2/2     Running            0          12m
zookeeper-operator-7f66975d9d-nfmkp                    1/1     Running            0          12m
```

### CODE REVIEW

File: ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- description: "Disk health is bad"
```
Comment (mishoyama) [inline ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml:1]: `Disk has failed`

File: ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name: "DiskFailure"
+  issueCategory: Auto
+  matchOnList:
+    - matchon:
+        - label: SymptomID
+          value: 2002
+        - field: type
+          value: Error
+  notifiers:
+    - objectscale-supportassist-ese
```
Comment (mishoyama) [inline ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml:11]: In OpenShift it should be UI and customer configured delivery channels if any (like remote syslog or email)

File: ecs-cluster/alerts/issues/2003_DISK_SUSPECT_issue.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+- description: "Disk health is suspect"
```
Comment (mishoyama) [inline ecs-cluster/alerts/issues/2003_DISK_SUSPECT_issue.yaml:1]: `Disk has suspected`

File: ecs-cluster/alerts/issues/2062_DISK_REPLACEMENT_FAILED_issue.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+- description: "Disk replacement procedure is failed"
```
Comment (mishoyama) [inline ecs-cluster/alerts/issues/2062_DISK_REPLACEMENT_FAILED_issue.yaml:1]: `Disk cannot be removed`

File: ecs-cluster/alerts/issues/2063_DISK_MISSING_issue.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+- description: "Disk is missed"
```
Comment (mishoyama) [inline ecs-cluster/alerts/issues/2063_DISK_MISSING_issue.yaml:1]: `Disk is missing`

File: ecs-cluster/alerts/remedies/2063_DISK_MISSING_remedies.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+- symptomid: 2063
+  description: Disk is missed
+  remedies:
+    - Check if disk is presented in the system
```
Comment (mishoyama) [inline ecs-cluster/alerts/remedies/2063_DISK_MISSING_remedies.yaml:4]: May be `Check system logs`?

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml (hunk 2 above)
Comment (AnastasiaShemyakinskaya) [reply ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml:11]: @eddiepavkovic, @vgupta-mickey  Could you please suggest the type of notifier for OpenShift?

Comment (mishoyama) [reply ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml:11]: May be `supportassist-objectscale`?

**Review by mishoyama — APPROVED** [review]

**Comment by mishoyama** [comment]
@ykulazhenkov @painhardcore please review

**Review by ykulazhenkov — APPROVED** [review]

**Review by painhardcore — APPROVED** [review]

File: ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+- description: "Disk has failed"
+  name: "DiskFailure"
+  issueCategory: Auto
+  matchOnList:
+    - matchon:
+        - label: SymptomID
+          value: 2002
+        - field: type
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/2002_DISK_FAILURE_issue.yaml:8]: you shouldn't use field (type) in the matching rule  this as the issue will not clear automatically when the event with type=Normal will come to clear the issue.

File: ecs-cluster/alerts/issues/2003_DISK_SUSPECT_issue.yaml [hunk 8]
```diff
@@ -0,0 +1,10 @@
+- description: "Disk has suspected"
+  name: "DiskSuspect"
+  issueCategory: Auto
+  matchOnList:
+    - matchon:
+        - label: SymptomID
+          value: 2003
+        - field: type
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/2003_DISK_SUSPECT_issue.yaml:8]: same

File: ecs-cluster/alerts/issues/2019_DISK_ADD_issue.yaml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+- description: "Disk has added"
+  name: "DiskAdded"
+  issueCategory: Auto
+  matchOnList:
+    - matchon:
+        - label: SymptomID
+          value: 2019
+        - field: type
```
Comment (vgupta-mickey) [inline ecs-cluster/alerts/issues/2019_DISK_ADD_issue.yaml:8]: same

**Review by vgupta-mickey — COMMENTED** [review]
We need to revert this and fix it right.

**Pull request merged**